### PR TITLE
Added default status and utilization for cluster with no volumes

### PR DESCRIPTION
**Pull request by shtripat**
In case there are no volumes in the cluster, gstatus doesnt return
the utilization and status details for the underlying cluster.

Added logic to calculate the status based on peer's connected status
and utilization to default to zeros.

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@nthomas-redhat plz review

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -267,6 +267,27 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+                # If gstatus does not return any status details in absence
+                # of volumes, default set the status and utilization details
+                out_lines = stdout.split('\n')
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: please provide sample of stdout

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 2]
```diff
@@ -267,6 +267,27 @@ def _run(self):
...
+                connected = True
+                for index in range(4, len(out_lines) - 2):
+                    if out_lines[index].split('\t')[2].strip() != 'Connected':
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: are you sure the stdout can always be processed this way?

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 1 above)
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: Sure. Will add one sample here.

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: I have seen output of gstatus is like this only always if volumes are not present.

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 3]
```diff
@@ -267,6 +267,40 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+
```
Comment (nthomas-redhat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: Are you running this code always even if the volumes are present? In that case there is a possibility of overwriting the status provided by gstatus?

Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: Ah it should be in else part of gstatus command. Will update

**Pull request merged**